lerna.json

```diff
@@ -1,6 +1,16 @@
 {
   "packages": [
-    "packages/*"
+    "packages/*",
+    "demos/html5/ckeditor4",
+    "demos/html5/froala2",
+    "demos/html5/froala3",
+    "demos/html5/generic",
+    "demos/html5/tinymce4",
+    "demos/html5/tinymce5",
+    "demos/html5/ckeditor5",
+    "demos/angular/*",
+    "demos/react/*"
```

Comment:
Is this change on purpose? Will this branch solve the issues we had with the Jenkins machines when running the tests?

Reply:
this branch solves the issue but these are not necessary, my mistake